src/main/services/repack-tracker/helpers.ts

```diff
@@ -5,7 +5,7 @@ import { repackRepository } from "@main/repository";
 
 import { QueryDeepPartialEntity } from "typeorm/query-builder/QueryPartialEntity";
 
-export const savePage = async (repacks: QueryDeepPartialEntity<Repack>[]) =>
+export const saveRepacks = async (repacks: QueryDeepPartialEntity<Repack>[]) =>
```

Comment:
is there any specific reason for changing the name of function in this pr?

Reply:
Yeah, sorry, I didn't specify this. But my goal with this change is to improve readability, as the function does not save any page, but instead saves the repacks on the database. While I was reading the code I found this suitable. I will enhance the description to contain this

Reply:
It does indeed save the page. The reason behind is that we keep the page column in order to track and resume from the page that the scraper left at if it crashes or simply cannot move any forward.

For now I think it's best to leave the name as it is.

![image](https://github.com/hydralauncher/hydra/assets/167036558/935e0366-4649-4c55-910f-4b4f96b1d3ad)

Reply:
Thank you for the information, reverting